Stable-Branch-Strategy.md

```diff
@@ -0,0 +1,103 @@
+Branch and release maintenance for Kata Containers project
+
+# Overview
+
+As detailed in the [release documentation](https://github.com/kata-containers/documentation/blob/master/Releases.md),
+the Kata Containers project makes use of semantic versioning.  A release version is described
+by MAJOR.MINOR.PATCH.  Early in the project, we will introduce many new features which will require
+minor version changes as well as bug fixes.  To facilitate a stable user environment, Kata will
+begin to provide stable branch based releases as well as a master branch release.
+
+## Stable branch patch criteria
+
+No new features should be introduced to stable branches.  The intention is to limit risk to users,
+providing only bug and security fixes.
+
+## Branch Management
+Kata Containers will maintain two stable release branches in addition to the master branch.
+Once a new MAJOR or MINOR release is created from master, a new stable branch will be created for
+the prior MAJOR or MINOR release and the older stable branch will no longer be maintained.
+
+A couple of examples are provided to help clarify this process.
+
+### New bug fix introduced
+
+A bug fix is submitted against the runtime which does not introduce new inter-component dependencies.
+This fix will be applied to both the master and stable branches, and there is no need to create a new
```

Comment:
This fix is applied to both the master and stable branches so there is no need to create a new